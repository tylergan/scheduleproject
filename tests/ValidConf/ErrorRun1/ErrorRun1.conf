at 0900 run /home/bob/myprog
on Saturday,Tuesday at 1654,1200 run /bin/echo hello