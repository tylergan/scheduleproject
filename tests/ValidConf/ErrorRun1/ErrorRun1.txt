Testing that the output for runner.py and runner.status works when: 
    1) The process specified to run in that configuration file will return a status not equal to 0
