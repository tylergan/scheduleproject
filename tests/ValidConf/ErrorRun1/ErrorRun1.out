Traceback (most recent call last):
  File "run_test_runner.py", line 179, in <module>
    os.execv(process.path, process.param)
FileNotFoundError: [Errno 2] No such file or directory
hello
hello
hello
hello
